### ISSUE TITLE
! Mass Breakage ! [several dozen pages] by [several Privacy filter lists] by [several filters]

### ISSUE THREAD

**Issue by felix-22** [easylist/easylist#9393]
### List the website(s) you're having issues:

`https://publicwww.com/websites/tdBlocksArray+%22COMMON-MIN.JS%22/`
Publicwww hides most names, could find only:

```
playjunkie.com
everydaykoala.com
5dwallpaper.com
wallpaperdata.com
sportzbonanza.com
tettybetty.com
```

### What happens?

Breakage, sites are not functional, thumbnails and images won't load

### List Subscriptions you're using:

Filter lists causing the breakage:
- EasyPrivacy
- AdGuard Enhanced Tracking
- uBlock filters – Privacy

### Your settings

- OS/version: W7SP164
- Browser/version: FF 93 / O 78
- Adblock Extension/version: 1.38.5b2

### Other details:

Nasty breakage caused by two different filters, needs double fix:

```
! unbreak for EasyPrivacy
@@||confiant-integrations.global.ssl.fastly.net^$script,domain=seeitlive.co|playjunkie.com|everydaykoala.com|5dwallpaper.com|wallpaperdata.com|sportzbonanza.com|tettybetty.com
@@||mxpnl.com^$domain=mixpanel.com|playjunkie.com|everydaykoala.com|5dwallpaper.com|wallpaperdata.com|sportzbonanza.com|tettybetty.com

```

`! unbreak for AdGuard Enhanced Tracking (to be found)`

```
! unbreak for uBlock Privacy
||taboola.com^$3p,domain=~5dwallpaper.com
```

### Screenshoots:
**Broken:**
![1](https://user-images.githubusercontent.com/75935068/137324294-89045044-f5c1-41e8-8a3d-70360b7ea6b5.png)

**Fixed:**
![2](https://user-images.githubusercontent.com/75935068/137324350-d5e27382-9293-42c3-82e3-4c5d8ccc7a28.png)

**Comment by peace2000** [comment]
@ryanbr could you test also with Firefox?

**Comment by ryanbr** [comment]
Using the suggested fixes.  
Simply refreshing the site over and over again would give me different results. Its either Firefox or the site CMS is here fault. 

https://gif.fanboy.co.nz/gif/ff-everyday.gif

Given this, i'll probably close this request. We can't fix this (simply refreshing the page with no changes, would possibly get different results)

**Comment by peace2000** [comment]
Yes it does give different results. But when I tested in a clean browser profile, no other extensions and uBO with default, filters provided by @felix-22 clearty reduced the prevalence of this issue for me.
